.rubocop_todo.yml

```diff
@@ -1,32 +1,41 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2015-10-14 13:50:41 -0700 using RuboCop version 0.34.2.
+# on 2017-02-25 23:23:21 +0900 using RuboCop version 0.46.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 4
+# Offense count: 5
+# Configuration parameters: Include.
+# Include: **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
+  Exclude:
+    - 'Gemfile'
```

Comment:
I can fix this Cop warning with editing Gemfile. But I do not know either this Cop should be enabled or not  in this repository. So I just added to `.rubocop_todo.yml` for now.